pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -416,11 +441,11 @@ func pathFiltersSingleLayerStr(c *VPCConfig, filterLayerName string, rules []Rul
 }
 
 // prints detailed list of rules that effects the (existing or non-existing) connection
-func (rulesInLayers rulesInLayers) rulesDetailsStr(c *VPCConfig, filtersRelevant map[string]bool, isIngress bool) string {
+func (rules rulesInLayers) rulesDetailsStr(c *VPCConfig, filtersRelevant map[string]bool, isIngress bool) string {
```

Comment:
"who is the receiver" comment

Reply:
Hmm... here I am not so sure this should not be the receiver. And in anyways, there are more cases in which the receiver is the "rules" that are the subject of the explainability; if we want to change this concept we should do it in a separate PR to be consistent.

Reply:
please open a relevant issue

Reply:
[OK, I'll open the issue.](https://github.com/np-guard/vpc-network-config-analyzer/issues/630)